package.json

```diff
@@ -26,6 +26,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
+    "react-native-credit-card-input": "^0.4.1",
```

Comment:
Please do not install this version of the app if you're running a newer version of Expo.

Instead install: 
`mobinni/react-native-credit-card-input#0.5.3`